README.md

```diff
@@ -57,7 +57,7 @@ The specification and code is licensed under the Apache 2.0 license found in the
 
 ## Code of Conduct
 
-Participation in the OpenContainers community is governed by [OpenContainer's Code of Conduct](code-of-conduct.md).
+Participation in the OpenContainers community is governed by [OpenContainer's Code of Conduct](https://github.com/opencontainers/tob/blob/master/code-of-conduct.md).
```

Comment:
@hqhq yes, sorry. The file does not strictly _need_ to be in the repo. Though any changes to the document in the tob repo, ought to be mirrored to the file in this repo.
While I like the idea of not needing to duplicate the work across repos, during the call I decided to leave this file in the repo for ease  and simplicity of making the PDF/HTML docs